### PR TITLE
Add testdata for BatchNorm3D layer

### PR DESCRIPTION
**Pull request by l-bat**

### CODE REVIEW

File: testdata/dnn/onnx/generate_onnx_models.py [hunk 1]
```diff
@@ -228,6 +232,91 @@ def forward(self, x):
...
+        conv = conv.permute(2, 0, 1)  # [w, b, c]
+
+        output = self.rnn(conv)
+        return output
+
+
+input = Variable(torch.randn(1, 1, 32, 128))
+model = CRNNNET(input.shape[2], 1, 128)
+save_data_and_model("LSTM", input, model)
+# output = np.ascontiguousarray(output.detach().numpy())
```
Comment (dkurt) [inline testdata/dnn/onnx/generate_onnx_models.py]: Can you please remove untested binaries and the code which generates them?
